conductr_cli/host.py

```diff
@@ -81,10 +82,9 @@ def is_listening(ip_addr, port):
 
 def addr_alias_setup_instructions(addrs, subnet_mask):
     def info_text(commands):
-        if len(commands) > 1:
-            return 'Run the following commands to create the address aliases on your machine:'
-        else:
-            return 'Run the following command to create an address alias on your machine:'
+        return headline(
```

Comment:
I would not use a headline here. Headlines should we used for certain sub sections to start ConductR, such as `Starting HAProxy`.